README.md

```diff
@@ -385,6 +456,117 @@ Installation Instructions
   ```
 
 
+JSON inventory method
+----------------------
+
+ There is also created Ansible playbook sample for deploying IBM Spectrum Scale (GPFS) cluster using json inventory.
```

Comment:
Why would anyone choose the JSON inventory method and which other methods are available?

My understanding is that e.g. Terraform produces inventory in JSON format... but for a user - are there any reasons to choose one over the other? Also, if this is generic Ansible functionality (and not specific to our code) then we may want to link to the official documentation for an explanation...